examples/src/main/scala/example/jsoniter/ExampleHttp4sApp.scala

```diff
@@ -0,0 +1,76 @@
+package example.jsoniter
```

Comment:
Yeah that's a good point. Changed it to zioHttp, although I only implemented the `httpRoutes` since the implementation of the websockets is quite lengthy.

On this note, would it be worth it to add methods in the adapters where the codecs can be implicitly provided instead? Or do you recon it's a relatively rare usecase for users to change the default JSON implementation?

Reply:
It's kind of historical, except for Akka the adapters were fixed to a particular Json library. Even if we exposed a method that took a codec instead, the adapters would still depend on a particular Json library so I imagine people wouldn't use them.